kong.conf.default

```diff
@@ -1222,6 +1222,30 @@
                                  #
                                  # See https://github.com/openresty/lua-nginx-module#lua_ssl_trusted_certificate
 
+#lua_ssl_trusted_certificate_combined =
```

Comment:
Is there a possibility we can reuse the existing `lua_ssl_trusted_certificate` config for this? It seems that the config format should be compatible with the existing `lua_ssl_trusted_certificate` format and that way we avoid any confusions from the user.

The idea is that `lua_ssl_trusted_certificate_combined` is generated and used by Kong internally, but it need not to be visible to the users.

Reply:
Yes, indeed! I didn't realize that we could rely on "hidden" (non-user-visible) config options.

This change involved modifying all the kong templates though. We'll have to document that.

In any case, I've re-done the PR using a hidden config option. Looks much better now, thanks!